core/src/main/scala/org/apache/spark/internal/config/History.scala

```diff
@@ -128,15 +128,19 @@ private[spark] object History {
 
   val CUSTOM_EXECUTOR_LOG_URL = ConfigBuilder("spark.history.custom.executor.log.url")
     .doc("Specifies custom spark executor log url for supporting external log service instead of " +
-      "using resource managers' app log urls. Spark will support some path variables via " +
-      "patterns which can be vary on resource managers. Please check the document for " +
-      "resource manager to refer which patterns are supported.")
+      "using cluster managers' application log urls in the history server. Spark will support " +
+      "some path variables via patterns which can vary on cluster manager. Please check the " +
+      "documentation for your each cluster manager to see which patterns are supported, if any. " +
```

Comment:
oops my fault -- "your each" should be just "your"

Reply:
No problem! Will fix.